### ISSUE TITLE
TFTrainer does not consider number of epochs when calculating learning rate

### ISSUE THREAD

**Issue by borisdayma** [huggingface/transformers#5045]
# 🐛 Bug
`TFTrainer` does not consider number of epochs when calculating learning rate

## Information

When using `TFTrainer`, learning rate decreases to 0 at the end of the first epoch, even when we want to train on multiple epochs.

The problem arises when using:
* [X] the official example scripts: (give details below)
* [ ] my own modified scripts: (give details below)

The tasks I am working on is:
* [X] an official GLUE/SQUaD task: run_tf_glue with mrpc task
* [ ] my own task or dataset: (give details below)

## To reproduce
`python run_tf_glue.py --model_name_or_path bert-base-cased --task_name MRPC --do_train --do_eval --max_seq_length 128 --per_device_train_batch_size 32 --learning_rate 2e-5 --num_train_epochs 3.0 --output_dir /tmp/test_hf/ --overwrite_output_dir --logging_dir hf --evaluate_during_training --eval_steps 50 --logging_steps 10`

This is what we get, where learning rate is null after the first epoch:
![image](https://user-images.githubusercontent.com/715491/84722988-39c6ec00-af4a-11ea-9c80-f2213d5a579c.png)

You can refer to [W&B run](https://app.wandb.ai/borisd13/huggingface/runs/2zcsfumy?workspace=user-borisd13) for more details.

## Expected behavior

Learning rate should slowly decrease until end of 3rd epoch.

## Environment info

     
- `transformers` version: 2.11.0
- Platform: Linux-5.3.0-53-generic-x86_64-with-glibc2.10
- Python version: 3.8.3
- PyTorch version (GPU?): N/A
- Tensorflow version (GPU?): 2.2.0 (True)
- Using GPU in script?: Yes (one only)
- Using distributed or parallel set-up in script?: No

@jplu I recorded the issue here so we don't forget to fix it.
Let me know if I can help.

**Comment by borisdayma** [comment]
This is fixed
